games-strategy/hedgewars/hedgewars-0.9.24.1.ebuild

```diff
@@ -41,7 +41,21 @@ CDEPEND="
 	!libav? ( media-video/ffmpeg:= )"
 DEPEND="${CDEPEND}
 	>=dev-lang/fpc-2.4
-	dev-qt/linguist-tools:5"
+	dev-qt/linguist-tools:5
+	server? (
+		>=dev-lang/ghc-6.10
```

Comment:
wait, it pulls in haskell for the server? that is insane 😱

Reply:
Yeah, that was my reaction when I first encountered that something like 10 years ago and that's even after learning Haskell at uni. :laughing: